code/src/test/java/com/google/sps/storage/BusinessTest.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.storage;
+
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+ 
+@RunWith(JUnit4.class)
+public final class BusinessTest {
+
+  @Test
+  public void createBusiness() {
+    Business business = new Business("Test");
+    Assert.assertFalse(business.contactedInLastWeek());
```

Comment:
nit: we generally want tests broken into three sections by whitespace: Arrange, Act, Assert. If Act and Assert are on the same line (as they are here) that's OK, but you still want space for the arrange block. I know this sounds funny, but it's actually REALLY helpful once you get into larger tests.

Reply:
okay